src/merge.js

```diff
@@ -0,0 +1,3 @@
+import merge from 'lodash.merge';
+
+export default merge;
```

Comment:
Sorry, I haven't catched up the conversation. So, you said
> how about lodash.assign? It uses Object.assign if it's available. and it's faster than extend or merge imo. @oliviertassinari Thoughts?

The solution I'm proposing is using the babel `_extend`. This babel function is using `Object.assign` when available or a polyfill. In my case, It was reducing the output size of the lib. The `_extend` was already needed for the spread operator syntax. In our case, `lodash.merge` is probably better :+1:.